snippets/unified-perspective-web-console.adoc

```diff
@@ -0,0 +1,22 @@
+// Snippet included in the following assemblies and modules:
+//
+// * getting-started/openshift-overview.adoc
+// * web_console/web-console-overview.adoc
+// * modules/about-administrator_web-console.adoc
+// * modules/about-developer_web-console.adoc
+// * modules/viewing-stats-collected-kubernetes-nmstate-op.adoc
+// * modules/viewing-a-project-using-the-web-console.adoc
+// * modules/red-hat-marketplace-features.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[IMPORTANT]
+====
+Starting with {product-title} 4.19, the perspectives in the web console have unified. There is no longer a *Developer* perspective by default.
+
+All {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission to certain features from the cluster owner.
+
+You can still enable the *Developer* perspective.
+See xref:../web_console/web-console-overview.adoc#enabling-developer-perspective_web-console_web-console-overview for more information on enabling the *Developer* perspective.
```

Comment:
Unless you've gotten an exception for this, xrefs are not allowed in snippets unfortunately: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#writing-text-snippets